src/libs/ajax/AzureStorage.ts

```diff
@@ -158,7 +159,14 @@ export const AzureStorage = (signal?: AbortSignal) => ({
         const textFileContents = await getObject()
         return fetchOk(`${getConfig().calhounUrlRoot}/${calhounPath}`,
           _.mergeAll([authOpts(), { signal, method: 'POST', body: textFileContents }])
-        ).then(res => res.text())
+        ).then(res => {
+          Ajax().Metrics.captureEvent(Events.analysisPreviewSuccess, { fileName: blobName, fileType: getExtension(blobName), cloudPlatform: cloudProviderTypes.AZURE })
```

Comment:
Some comment about using Ajax here as with GoogleStorage. Somehow javascript magically handles the circular dependency, but it starts complaining when you start adding tests! I know someone else pulled it in for this module already, and I'm not necessarily saying we fix it in this PR(though it would be really nice to move the submodule within Ajax that requires this out and remove the cycle....), but at least don't deepen this issue and pull in metrics directly! cc @msilva-broad and I did a good bit of work to remove these cycles!